electrum/wallet.py

```diff
@@ -1747,19 +1747,26 @@ def fee_estimator(size: Union[int, float, Decimal]) -> int:
                     lower_bound = lower_bound if not is_local else 0
                     return int(max(lower_bound, original_fee_estimator(size)))
                 txi = base_tx.inputs()
-                txo = list(filter(lambda o: not self.is_change(o.address), base_tx.outputs()))
+                txo = list(filter(lambda o: not self.is_change(o.address), base_tx.outputs())) + list(outputs)
+                output_dict = {}
+                for output in txo:
+                    if output.address in output_dict:
```

Comment:
Should use `output.scriptpubkey` instead, as not all scripts have a corresponding `address` in which case the value would be None.

Reply:
https://github.com/spesmilo/electrum/pull/8474/commits/f5a58675f131140de60c38a43194bdd5d6fdde1d